services/svc-pinger/test/url.scheme.test.ts

```diff
@@ -0,0 +1,14 @@
+import { describe, it, expect } from 'vitest'
+import { urlScheme } from '../src/schemes/url.scheme'
+
+const invalidAddresses = ["1:2", "zxc", "123"]
+const validAddresses = ["example.com", "localhost:3000", "sub.domain.org", "www.google.com/search?q=test", "example.org"]
+
+describe("URL scheme", () => {
```

Comment:
To avoid unnecessary duplication of tests, the schema should be tested within the integration tests of corresponding endpoints